addons/calendar/tests/test_event_notifications.py

```diff
@@ -124,7 +124,7 @@ def test_bus_notif(self):
                 "alarm_id": alarm.id,
                 "event_id": self.event.id,
                 "title": "Doom's day",
-                "message": self.event.display_time,
+                "message": self.event.display_time + " ",
```

Comment:
Still not sure what that does?

Reply:
In the first place, it was to reflect the additional part for the message when adding the body for a notification. But since I've reformatted the message, it is no longer of use.